cpp-client/deephaven/dhcore/include/public/deephaven/dhcore/utility/utility.h

```diff
@@ -187,6 +187,16 @@ TimePointToEpochMillis(
 TimePointToStr(
     std::chrono::time_point<std::chrono::system_clock> time_point);
 
+/**
+ * This is a method that simply invokes std::filesystem::path(path).filename().string();
+ * We put it here because it is sometimes useful, to provide functionality
+ * similar to the POSIX basename() call. We deliberately do not inline it
+ * because is generates a surprising amount of code.
+ * @param path The path
+ * @return The filename of the path, as returned by std::filesystem::path(path).filename().string()
+ */
+std::string GetFilename(std::string_view path);
```

Comment:
Can we call it Basename?  I think Basename is unambiguous; I believe filename as a noun doesn't imply with or without a full path.

Reply:
ok done